staff/staff_views_helper.py

```diff
@@ -358,3 +411,115 @@ def get_integer_days(class_offering):
         return [0, 2]
     elif class_offering.meeting_days == "T/R":
         return [1, 3]
+
+
+def get_course_attendance_statistic(course_id):
+    class_attendance = Attendance.objects.filter(
+        classroom_id=course_id, date__range=["2000-01-01", datetime.today().date()]
+    )
+    average = (
+        sum(
+            attendance_object.presence == "Present"
+            or attendance_object.presence == "Late"
+            for attendance_object in list
```

Comment:
I don't think this code will work, `list` doesn't appear to be a bound variable in here, so it's referring to the global type. Probably should be `class_attendance`?

Reply:
Correct, that wouldn't have worked... my excuse is that I must have been tired that day....  Fixed in c62a17f4efff8ca9abf14bc84c1c5547650429ae